### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by fchensan**
Add an FAQ section to the homepage.

### CODE REVIEW

File: src/routes/index.svelte [hunk 1]
```diff
@@ -267,3 +324,30 @@
...
+            No worries! Most of our members have little to no experience with angklung prior to joining the ensemble, but everyone is keeping up because angklung is a very beginner-friendly instrument!
+            <br><br>
+            <span class="question">Do I need to own an angklung?</span><br>
+            No, you don’t! Angklungs will be provided to all members.
+            <br><br>
+            <span class="question">What are some of the activities?</span><br>
+            Our highlight of the year is the annual performance for CAC+US. In addition to that, we also have our fun-filled Angklung Camp where we spend two days playing games and doing activities!
+            <br><br>
+            <span class="question">Who can I contact?</span><br>
+            You can reach us via email at nus.angklung@gmail.com. Do also check out our Instagram account: @nusangklung, Facebook page: NUS Angklung Ensemble, and Youtube channel: NUS Angklung Ensemble.
```
Comment (benclmnt) [inline src/routes/index.svelte]: Instead of putting email gini, suru contact aja pake contact form dibawah, sama kasi link ke contact page kita aja

File: src/routes/index.svelte [hunk 2]
```diff
@@ -267,3 +324,30 @@
...
+            We welcome all interested NUS undergraduate and graduate students.
+            <br><br>
+            <span class="question">When are the practice sessions and what if my schedule does not allow me to attend the practice sessions?</span><br>
+            We have practice sessions every Monday and Wednesday from 19:00 to 21:00. You only need to attend one of these weekly sessions so you can choose which one to attend depending on your schedule.
+            <br><br>
+            <span class="question">What if I do not have any musical background or experience?</span><br>
+            No worries! Most of our members have little to no experience with angklung prior to joining the ensemble, but everyone is keeping up because angklung is a very beginner-friendly instrument!
+            <br><br>
+            <span class="question">Do I need to own an angklung?</span><br>
+            No, you don’t! Angklungs will be provided to all members.
```
Comment (benclmnt) [inline src/routes/index.svelte]: will be lent*

File: src/routes/index.svelte [hunk 3]
```diff
@@ -267,3 +324,30 @@
...
+            <br><br>
+            <span class="question">What are some of the activities?</span><br>
+            Our highlight of the year is the annual performance for CAC+US. In addition to that, we also have our fun-filled Angklung Camp where we spend two days playing games and doing activities!
```
Comment (benclmnt) [inline src/routes/index.svelte]: Ini kita udah ada pagenya di /about. Ga perlu ada lagi. Ato minta brilly ubah konten di page itu aja.

File: src/routes/index.svelte [hunk 4]
```diff
@@ -37,7 +37,7 @@
     }
 
     .angklung-logo::after {
-        content: '';
+        answer: '';
```
Comment (benclmnt) [inline src/routes/index.svelte]: Why is this changed?

**Review by benclmnt — COMMENTED** [review]
Looks good except that one part

**Review by benclmnt — APPROVED** [review]

File: src/routes/index.svelte (hunk 4 above)
Comment (fchensan) [reply src/routes/index.svelte]: I think `answer` is a more descriptive class name (but I'm force pushing some changes which removes this change) 😅
